rhizome/host/lib/vm_setup.rb

```diff
@@ -551,7 +551,8 @@ def download_boot_image(boot_image)
     urls = {
       "ubuntu-jammy" => "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img",
       "almalinux-9.1" => "https://repo.almalinux.org/almalinux/9/cloud/x86_64/images/AlmaLinux-9-GenericCloud-latest.x86_64.qcow2",
-      "opensuse-leap-15.4" => "https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2"
+      "opensuse-leap-15.4" => "https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-Minimal-VM.x86_64-OpenStack-Cloud.qcow2",
+      "github-ubuntu-2204" => nil
```

Comment:
I'm downloading them manually for now. Since we will have dedicated VmHost for GitHub runner, it's manageable for now.

We need to automate building golden image and downloading it.

I will create another PR for downloading logic.
It's 86GB and downloading it with `curl` isn't efficient. If we store image at Azure Blob Storage, `azcopy` downloads it faster.

Also we can't share image publicly because of legal issues. We need to hide it back of a config value.